sqlobject/src/test/java/org/jdbi/v3/sqlobject/AdvancableClock.java

```diff
@@ -16,27 +16,32 @@
 import java.time.Clock;
 import java.time.Instant;
 import java.time.ZoneId;
+import java.time.ZonedDateTime;
 import java.time.temporal.TemporalUnit;
 
-class MockClock extends Clock {
-    private Instant now = Instant.now();
+public class AdvancableClock extends Clock {
```

Comment:
How come this class name changed?  This seems unrelated to the meat of this PR.  It seems unrelated to time handling?

Reply:
Just a tiny collateral improvement, came across it somehow and thought it needed a spit shine. Instead of throwing exceptions and assuming system timezone data, it should just take up the timezone as part of its function. Turns it into a functional component instead of just a hack to get around something. I can take the change out if it bothers you but it's so small it may as well just be merged along ¯\\\_(ツ)_\/¯